aiofreepybox/api/sharelink.py

```diff
@@ -0,0 +1,33 @@
+class Sharelink:
+    """
+    Sharelink
+    """
+
+    def __init__(self, access):
+        self._access = access
+
+    share_link_data_schema = {"path": "", "expire": 0}
```

Comment:
```suggestion
    share_link_data_schema = {"path": "", "expire": 0, "fullurl": ""}
```

Reply:
i've also changed it to take path and expire on input : create_share_link(self, path, expire)